packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks.ts

```diff
@@ -62,8 +62,8 @@ const stockLineToDraftLine = (
 ): DraftStocktakeLine => {
   return {
     stocktakeId,
-    countThisLine: false,
-    isCreated: false,
+    countThisLine: true,
+    isCreated: true,
```

Comment:
default to `isCreated` for lines with a stocktake line. If the user doesn't want to count it, then `countThisLine` will be false.

using `countThisLine` as true is questionable. It means when you add an item, you're counting all the stock lines in the store by default.. 🤔